### PR TITLE
Chore/rr improvements

### PR DESCRIPTION
**Pull request by joshenlim**

### CODE REVIEW

File: apps/studio/components/interfaces/Settings/Infrastructure/InfrastructureConfiguration/MapView.tsx [hunk 1]
```diff
@@ -182,14 +181,16 @@ const MapView = ({
...
+                      hasNoDatabases ? 'fill-black/30 dark:fill-white/30' : 'fill-brand'
+                    }`}
                   />
                 )}
                 <circle
                   r={4}
                   className={`cursor-pointer ${
                     hasNoDatabases
-                      ? 'fill-background-surface-300 stroke-white/20'
+                      ? 'fill-background-surface-300 stroke-black/20 dark:stroke-white/20'
```
Comment (MildTomato) [inline apps/studio/components/interfaces/Settings/Infrastructure/InfrastructureConfiguration/MapView.tsx:193]: you can use `stroke-border-**` btw

File: apps/studio/components/interfaces/Settings/Infrastructure/InfrastructureConfiguration/MapView.tsx [hunk 2]
```diff
@@ -198,36 +199,37 @@ const MapView = ({
               </Marker>
             )
           })}
+
+          {tooltip !== undefined && zoom === 1.5 && (
+            <Marker coordinates={[tooltip.x - 47, tooltip.y - 5]}>
+              <foreignObject width={220} height={66.25}>
+                <div className="bg-background bg-opacity-50 rounded border">
```
Comment (MildTomato) [inline apps/studio/components/interfaces/Settings/Infrastructure/InfrastructureConfiguration/MapView.tsx:206]: `bg/50` shorthand if you need

File: apps/studio/components/interfaces/Settings/Infrastructure/InfrastructureConfiguration/MapView.tsx [hunk 3]
```diff
@@ -182,14 +181,16 @@ const MapView = ({
                 {selectedRegionKey === region.region && (
                   <circle
                     r={4}
-                    className={`animate-ping ${hasNoDatabases ? 'fill-white/30' : 'fill-brand'}`}
+                    className={`animate-ping ${
+                      hasNoDatabases ? 'fill-black/30 dark:fill-white/30' : 'fill-brand'
```
Comment (MildTomato) [inline apps/studio/components/interfaces/Settings/Infrastructure/InfrastructureConfiguration/MapView.tsx:185]: probably need `bg-alternative/30`

Comment (MildTomato) [reply apps/studio/components/interfaces/Settings/Infrastructure/InfrastructureConfiguration/MapView.tsx:185]: sorry, `fill-background-alternative/30`

File: apps/studio/components/interfaces/Settings/Infrastructure/InfrastructureConfiguration/MapView.tsx (hunk 1 above)
Comment (joshenlim) [reply apps/studio/components/interfaces/Settings/Infrastructure/InfrastructureConfiguration/MapView.tsx:193]: i don't think its working it seems
image

Comment (joshenlim) [reply apps/studio/components/interfaces/Settings/Infrastructure/InfrastructureConfiguration/MapView.tsx:193]: ahh okay - i need to figure out what stroke-border-** are available
stroke-border-stronger works perhaps

File: apps/studio/components/interfaces/Settings/Infrastructure/InfrastructureConfiguration/MapView.tsx (hunk 3 above)
Comment (joshenlim) [reply apps/studio/components/interfaces/Settings/Infrastructure/InfrastructureConfiguration/MapView.tsx:185]: opted for `fill-border-stronger`, `fill-background-alternative` is a little too dark on dark mode

**Review by alaister — APPROVED** [review]
LGTM!

**Pull request merged**